doc/user/radial_and_axial_expansion.rst

```diff
@@ -2,11 +2,10 @@
 Radial and Axial Expansion and Contraction
 ******************************************
 
-ARMI natively supports linear expansion in both the radial and axial dimensions.
-These expansion types function independently of one another and each have their
-own set of underlying assumptions and use-cases. The remainder of this section
-is described as follows: in Section :ref:`thermalExpansion` the methodology used
-for thermal expansion within ARMI is described.
+ARMI natively supports linear expansion in both the radial and axial dimensions. These expansion
+types function independently of one another and each have their own set of underlying assumptions
+and use-cases. The remainder of this section is described as follows: in Section
+:ref:`thermalExpansion` the methodology used for thermal expansion within ARMI is described.
```

Comment:
This sentence is weird. It looks like it is going to describe a list of things, but then just lists one single section.

Reply:
Oh, eww. Removed. Thanks!